lib/experimental/Widgets/Content/CopyLabel/index.tsx

```diff
@@ -0,0 +1,33 @@
+import { Icon } from "@/components/Utilities/Icon"
+import { LayersFront } from "@/icons"
+import { forwardRef } from "react"
+interface CopyLabelType {
+  text: string
+}
+
+export const CopyLabel = forwardRef<HTMLDivElement, CopyLabelType>(
+  ({ text }, ref) => {
+    const copyHandler = async () => {
+      try {
+        await navigator.clipboard.writeText(text)
```

Comment:
At the minimum least, this deserves a [Toast](https://ui.shadcn.com/docs/components/toast) or a [Sonner](https://ui.shadcn.com/docs/components/sonner). We don't have that element yet, but we do need some kind of feedback. I would consider doing both at the same time.

But then... If we have "toasts" we need to think of other use cases like this one.

Reply:
we can also show a hint or change the icon to "copied!". we need a design decision here